squabble/cli.py

```diff
@@ -105,8 +105,9 @@ def run_linter(base_config, paths, expanded):
 
     issues = []
     for file_name, contents in files:
-        file_config = config.apply_file_config(base_config, contents)
-        if file_config is None:
+        file_config = config.apply_file_config(base_config, file_name, contents)
+        # The file should be skipped if no specific line is ignored
+        if file_config.skip.get(file_name) == []:
```

Comment:
So if I'm reading this right, line-based skips would take precedence over skipping the entire file? As a user, I think my expectation would be the opposite